.gitignore

```diff
@@ -43,6 +43,8 @@ vagrantconfig_local.yaml
 vagrant/manifests/classes/custom.pp
 node_modules
 .npm/
+static/css/node_lib/*
+static/js/node_lib/*
```

Comment:
I wonder actually would it make sense to commit the copied deps? That would then make us less dependent on node-dep changes in the builds changing unexpectedly. Or alternatively we should look at using yarn and committing the lock file and verifying the yarn.lock as part of the deploy process.